path_planning/src/path_planning.cpp

```diff
@@ -160,15 +160,15 @@ class planner {
 					status = kino_path_finder_->search(start_pt, start_vel, start_acc, goal_pt, goal_vel, false);
 
 					if (status == KinodynamicAstar::NO_PATH) {
-					std::cout << "[kino replan]: Can't find path." << std::endl;
-					return;
+						// std::cout << "[kino replan]: Can't find path." << std::endl;
```

Comment:
I actually do want to see this, but maybe we can change it to ROS_INFO_THROTTLE? This is helpful to me when debugging a flight but it does junk up the log ^_^